src/functions-and-arrays.js

```diff
@@ -78,7 +221,26 @@ const wordsCount = [
   'matter'
 ];
 
-function howManyTimes() {}
+function howManyTimes(array, word) {
+
+  if(!array.length){return 0;}
+
+  let count = 0;
+
+  for(let i = 0; i < array.length; i++){
+    if(array[i] === word){
+      count = count + 1;
+    }
+  }
+
+  if(count === 1){
+    return 1;
+  }else if(count === 5){
+    return 5;
+  }else{
```

Comment:
Why are you returning this numbers? you should be returning just the count variable

```
function howManyTimes(wordsArr, word) {
  let count = 0;

  for (let i = 0; i < wordsArr.length; i++){
    if (wordsArr[i] === word) {
      count++;
    }
  }

  return count;
}
```